src/test/java/pl/touk/sputnik/HttpConnectorEnv.java

```diff
@@ -0,0 +1,29 @@
+package pl.touk.sputnik;
+
+import com.github.tomakehurst.wiremock.client.UrlMatchingStrategy;
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpStatus;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+
+public class HttpConnectorEnv {
+
+    protected void stubGet(UrlMatchingStrategy url, String responseFile) throws Exception {
+        stubFor(get(url)
+                .withHeader("Authorization", equalTo("Basic dXNlcjpwYXNz"))
+                .willReturn(aResponse()
+                        .withStatus(HttpStatus.SC_OK)
+                        .withHeader("Content-Type", "application/json")
+                        .withBody(IOUtils.toString(getClass().getResourceAsStream(responseFile)))));
```

Comment:
[FindBugs] WARNING: UI: Usage of GetResource in pl.touk.sputnik.HttpConnectorEnv.stubGet(UrlMatchingStrategy, String) may be unsafe if class is extended